--- a/layout.mobintl.h
+++ b/layout.mobintl.h
@@ -234,7 +234,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.5, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {",", "'", 1.0, Code, KEY_COMMA, &layouts[ComposeMath]},
   {"", "Tab", 4.0, Code, KEY_SPACE},
   {".", "?", 1.0, Code, KEY_DOT, &layouts[ComposePunctuation]},
@@ -308,7 +308,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.5, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {",", "'", 1.0, Code, KEY_COMMA, &layouts[ComposeMath]},
   {"", "Tab", 4.0, Code, KEY_SPACE},
   {".", "?", 1.0, Code, KEY_DOT, &layouts[ComposePunctuation]},
@@ -467,7 +467,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {"Ctr", "Ctr", 1.0, Mod, Ctrl, .scheme = 1},
   {"-", "_", 0.75, Code, KEY_MINUS, &layouts[ComposeMath]},
   {",", "'", 0.75, Code, KEY_COMMA, &layouts[ComposeBracket]},
@@ -563,7 +563,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {",", "'", 1.0, Code, KEY_EQUAL, &layouts[ComposeMath]},
   {"", "Tab", 4.0, Code, KEY_SPACE},
   {".", "?", 1.0, Code, KEY_SLASH, &layouts[ComposePunctuation]},
@@ -628,7 +628,7 @@
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
   {"⇧", "⇫", 1.0, Mod, Shift, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {"", "Tab", 5.0, Code, KEY_SPACE},
   {"Enter", "Enter", 2.0, Code, KEY_ENTER, .scheme = 1},
 
@@ -688,7 +688,7 @@
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
   {"⇧", "⇫", 1.0, Mod, Shift, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {"", "Tab", 5.0, Code, KEY_SPACE},
   {"Enter", "Enter", 2.0, Code, KEY_ENTER, .scheme = 1},
 
@@ -752,7 +752,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {".", "،", 1.0, Code, KEY_DOT, &layouts[ComposePunctuation]},
   {"", "Tab", 4.0, Code, KEY_SPACE},
   {"⥣", "↥", 1.0, Code, KEY_GRAVE},
@@ -863,7 +863,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {",", "\"", 1.0, Code, KEY_APOSTROPHE, &layouts[ComposeMath]},
   {"", "Tab", 3.5, Code, KEY_SPACE},
   {".", "?", 1.0, Code, KEY_SLASH, &layouts[ComposePunctuation]},
@@ -1150,7 +1150,7 @@
   {"", "", 0.0, EndRow},
 
   {"⌨͕", "⌨͔", 1.0, NextLayer, .scheme = 1},
-  {"Cmp", "Cmp", 1.0, Compose, .scheme = 1},
+  {"Sup", "Sup", 1.0, Mod, Super, .scheme = 1},
   {"Ctr", "Ctr", 1.0, Mod, Ctrl, .scheme = 1},
   {",", "'", 1.0, Code, KEY_COMMA, &layouts[ComposeMath]},
   {"", "Tab", 5.0, Code, KEY_SPACE},
